pkg/interop/storage/storage.go

```diff
@@ -14,6 +14,20 @@ import "github.com/nspcc-dev/neo-go/pkg/interop/iterator"
 // to Neo .net framework's StorageContext class.
 type Context struct{}
 
+// FindFlags represents parameters to `Find` iterator.
+type FindFlags byte
+
+const (
+	// None is default option.
+	None FindFlags = 0
+	// KeysOnly is used for iterating over keys.
+	KeysOnly FindFlags = 1 << 0
+	// RemovePrefix is used for stripping prefix from keys.
+	RemovePrefix FindFlags = 1 << 1
+	// ValuesOnly is used for iterating over values.
+	ValuesOnly FindFlags = 1 << 2
```

Comment:
Can we create a test for these flags to check that they match the flags from `core` package? Like the one we had for smartcontract's triggers, it would help not to forget about them if we're to change something in `core`'s flags.

Reply:
The test is here, in the same commit (`TestFindFlags`)

Reply:
Somehow I missed that, sorry.